static/js/controllers/guided-setup-modal.js

```diff
@@ -0,0 +1,151 @@
+var libphonenumber = require('libphonenumber/utils'),
+    countries = require('../modules/countries');
+
+// TODO convert this controller to use angular more and jquery less
```

Comment:
can you file an issue for the TODO? So that it doesn't get forgotten for ever. (and link url here)